steps/bitrise-step-xctest-cobertura-xml/1.0.0/step.yml

```diff
@@ -0,0 +1,64 @@
+title: XCTest result to Cobertura XML
```

Comment:
Would recommend the title `Convert XCTest result to Cobertura XML`, to include the action done.

Reply:
Sure, thanks!